_tools/pydantic.md

```diff
@@ -0,0 +1,10 @@
+---
+title: OSCAL-Pydantic-v2
+author: Credentive Security
+license: CC0 1.0
+link: https://github.com/RS-Credentive/oscal-pydantic/tree/oscal-pydantic-v2
+---
+
+A python library designed to produce strictly conformant OSCAL models with a modern, 
+highly pythonic API. The tool leverages Pydantic for run-time type enforcement. 
+Just ‘pip install oscal-pydantic-v2’
```

Comment:
```suggestion
```

Want to conform this description to the other tool descriptions, which only include the general info about the app. I think we should remove the last sentence including installation instructions - this can just be found in the `README.md` of the repo.